src/bt-core.c

```diff
@@ -70,7 +75,18 @@ init_bt_core(bt_callbacks_t* callbacks)
     goto err_bt_interface_init;
   }
 
+#if ANDROID_VERSION >= 21
+  status = bt_interface->set_os_callouts(callouts);
```

Comment:
That's actually a good example of why we want to move Bluedroid out of Gecko. :(